presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -197,38 +197,45 @@ export class PresentationRpcInterface extends RpcInterface {
     NOTE: Please consult the README in core/common/src/rpc for the semantic versioning rules.
   ===========================================================================================*/
 
+  /** @deprecated in 4.10. Use [PresentationManager]($presentation-backend) instead of calling the RPC interface directly. */
   public async getNodesCount(_token: IModelRpcProps, _options: HierarchyRpcRequestOptions): PresentationRpcResponse<number> {
     return this.forward(arguments);
   }
 
+  /** @deprecated in 4.10. Use [PresentationManager]($presentation-backend) instead of calling the RPC interface directly. */
```

Comment:
```suggestion
  /** @deprecated in 4.10. Use [PresentationManager]($presentation-frontend) instead of calling the RPC interface directly. */
```

This suggestion applies to all similar deprecation messages in this file.

Reply:
fixed